website/addons/s3/static/s3NodeConfig.js

```diff
@@ -191,6 +191,13 @@ ViewModel.prototype.importAuth = function() {
 ViewModel.prototype.createCredentials = function() {
     var self = this;
     self.creatingCredentials(true);
+
+    if ( !(self.secretKey() && self.accessKey()) ){
+        var message = 'All those fields above are required.';
```

Comment:
@sloria  Yes, adding required is one way to do it. However I find required attribute is not supported in safari. And also input `required` is not used in most of our current implementation.
![screenshot 2015-07-29 15 11 28](https://cloud.githubusercontent.com/assets/5420789/8967334/30ebfd42-3604-11e5-967e-e1b8dbae5ac2.png)